options/preprocess.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto2";
```

Comment:
Please change to `syntax = "proto3"` as we try to stick to proto3 syntax for infoblox plugins. I also see that you use `proto3` for the demo.

Reply:
ok